website/content/docs/platform/k8s/helm/examples/kubernetes-auth.mdx

```diff
@@ -14,33 +14,18 @@ In this example, we will walk through how to set up the [Kubernetes Auth Method]
 
 This assumes the following commands will be run inside a Vault pod running in Kubernetes.
 
-You will optionally need the following variables:
-
-```bash
-# JWT is a service account token that has access to the Kubernetes TokenReview API
-# You can retrieve this from inside a pod at: /var/run/secrets/kubernetes.io/serviceaccount/token
-JWT=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
-
-# Address of Kubernetes itself as viewed from inside a running pod
-KUBERNETES_HOST=https://${KUBERNETES_PORT_443_TCP_ADDR}:443
-
-# Kubernetes internal CA
-KUBERNETES_CA_CERT=$(cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt)
-```
-
 Exec into the Vault pod:
 
 ```bash
-kubectl exec -it vault-0 /bin/sh
+kubectl exec -it vault-0 -- /bin/sh
 ```
 
-Then run the following command to configure the Kubernetes Auth Method:
+First, login using the `root` token and then run the following commands to configure the Kubernetes Auth Method:
```

Comment:
```suggestion
If you didn't set `server.dev.enabled=true`, you'll need to log in to Vault first using `vault login`. Then run the following commands to configure the Kubernetes Auth Method:
```

Reply:
@tomhjp Thank you for the review. I've fixed according to comments.